frontend/src/components/Login.jsx

```diff
@@ -0,0 +1,68 @@
+import { signInWithEmailAndPassword } from "firebase/auth";
+import React, { useState } from "react";
+import { auth } from "../config/firebase-config";
+import { toast } from "react-toastify";
+// import GoogleAuth from "./GoogleAuth";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      await signInWithEmailAndPassword(auth, email, password);
+      console.log("User logged in Successfully");
+      window.location.href = "/profile";
+      toast.success("User logged in Successfully", {
+        position: "top-center",
+      });
+    } catch (error) {
+      console.log(error.message);
+
+      toast.error(error.message, {
```

Comment:
is there a way to show the human readable string instead? 
<img width="345" alt="Screenshot 2024-07-08 at 09 06 04" src="https://github.com/devintor/breadbox/assets/3680271/c4b44b11-0f54-4fe7-9e23-d42d687d1ebd">